lib/tokens/session_token.js

```diff
@@ -56,14 +49,7 @@ module.exports = (log, Token, config) => {
       }
     }
 
-    // Parse the user agent string, then check the result to see whether
-    // the session token needs updating.
-    //
-    // If the session token has not changed, allowing up to an hour of
-    // leeway for lastAccessTime, return false.
-    //
-    // Otherwise, update properties on this then return true.
-    //
+    // Parse the user agent string, then update properties on this
     // It is the caller's responsibility to update the database.
     update(userAgentString) {
```

Comment:
Definitely not for now but fwiw, in a future refactoring I suspect we may get rid of this method and just call `userAgent` from the route handler directly. It feels like an unnecessary indirection after these changes.